Mapping/MappingDriver.php

```diff
@@ -56,4 +56,12 @@ public function loadMetadataForClass($className, ClassMetadata $metadata): void
         $metadata->setCustomGeneratorDefinition(['instance' => $idGenerator] + $metadata->customGeneratorDefinition);
         $metadata->setIdGeneratorType(ClassMetadataInfo::GENERATOR_TYPE_NONE);
     }
+
+    /**
+     * Returns driver chain
```

Comment:
I would say `Returns the inner driver.` instead. There is nothing guaranteeing that it is a driver chain.